<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Special Effect Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="shadowTab">
      <attribute name="title">
       <string>Shadow</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="verticalSpacing">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="shadow">
         <property name="text">
          <string>Enable client-side shadows on windows</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="no_dock_shadow">
         <property name="text">
          <string>Avoid drawing shadows on dock/panel windows</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="no_dnd_shadow">
         <property name="text">
          <string>Don't draw shadows on DND windows</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="clear_shadow">
         <property name="toolTip">
          <string>Fix some weirdness with ARGB windows</string>
         </property>
         <property name="text">
          <string>Zero the part of the shadow's mask behind the window</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="shadow_radius_label">
         <property name="text">
          <string>The blur radius for shadows</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="shadow_radius"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="shadow_offset_x_label">
         <property name="text">
          <string>The left offset for shadows</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="shadow_offset_x">
         <property name="minimum">
          <number>-99</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="shadow_offset_y_label">
         <property name="text">
          <string>The top offset for shadows</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="shadow_offset_y">
         <property name="minimum">
          <number>-99</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="shadow_opacity_label">
         <property name="text">
          <string>The translucency for shadows</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDoubleSpinBox" name="shadow_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Color of shadow</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="shadow_color">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QCheckBox" name="shadow_ignore_shaped">
         <property name="text">
          <string>Avoid drawing shadow on all shaped windows</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QCheckBox" name="xinerama_shadow_crop">
         <property name="text">
          <string>Crop shadows of maximized windows from extended displays</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="opacityTab">
      <attribute name="title">
       <string>Opacity</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="menu_opacity_label">
         <property name="text">
          <string>The opacity for menus</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="inactive_opacity_label">
         <property name="text">
          <string>Default opacity of inactive windows</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="active_opacity_label">
         <property name="text">
          <string>Default opacity for active windows</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="frame_opacity_label">
         <property name="text">
          <string>Opacity of window titlebars and borders</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="alpha_step_label">
         <property name="text">
          <string>XRender backend: Step size for alpha pictures
(Increasing it may result in less X resource usage)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="inactive_dim_label">
         <property name="text">
          <string>Dim inactive windows</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="inactive_dim_fixed">
         <property name="text">
          <string>Do not let dimness adjust based on window opacity</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QCheckBox" name="blur_background">
         <property name="text">
          <string>Blur background of transparent windows</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QCheckBox" name="blur_background_frame">
         <property name="text">
          <string>Blur background of opaque windows with transparent frames as well</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QCheckBox" name="blur_background_fixed">
         <property name="text">
          <string>Do not let blur radius adjust based on window opacity</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="active_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="inactive_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="menu_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="frame_opacity">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="alpha_step">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="inactive_dim">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.050000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fadeTab">
      <attribute name="title">
       <string>Fade</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="fading">
         <property name="text">
          <string>Fade windows during opacity changes</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="fade_delta_label">
         <property name="text">
          <string>The time between steps in a fade in milliseconds</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="fade_in_step_label">
         <property name="text">
          <string>Opacity change between steps while fading in</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="fade_out_step_label">
         <property name="text">
          <string>Opacity change between steps while fading out</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="no_fading_openclose">
         <property name="text">
          <string>Avoid fade windows in/out when opening/closing</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="fade_in_step">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.028000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="fade_out_step">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.030000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="fade_delta"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="otherTab">
      <attribute name="title">
       <string>Other</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="1">
        <widget class="QGroupBox" name="backend">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Rendering backend</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="backend_xrender">
            <property name="text">
             <string>X Render</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="backend_glx">
            <property name="text">
             <string>GLX (OpenGL)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>shadow</tabstop>
  <tabstop>no_dock_shadow</tabstop>
  <tabstop>no_dnd_shadow</tabstop>
  <tabstop>clear_shadow</tabstop>
  <tabstop>shadow_radius</tabstop>
  <tabstop>shadow_offset_x</tabstop>
  <tabstop>shadow_offset_y</tabstop>
  <tabstop>shadow_opacity</tabstop>
  <tabstop>shadow_color</tabstop>
  <tabstop>buttonBox_1</tabstop>
  <tabstop>menu_opacity</tabstop>
  <tabstop>inactive_opacity</tabstop>
  <tabstop>active_opacity</tabstop>
  <tabstop>frame_opacity</tabstop>
  <tabstop>alpha_step</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>inactive_dim</tabstop>
  <tabstop>inactive_dim_fixed</tabstop>
  <tabstop>blur_background</tabstop>
  <tabstop>blur_background_frame</tabstop>
  <tabstop>blur_background_fixed</tabstop>
  <tabstop>fading</tabstop>
  <tabstop>fade_delta</tabstop>
  <tabstop>no_fading_openclose</tabstop>
  <tabstop>fade_in_step</tabstop>
  <tabstop>fade_out_step</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_1</sender>
   <signal>rejected()</signal>
   <receiver>MainDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_1</sender>
   <signal>accepted()</signal>
   <receiver>MainDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
